docs/examples/Station.ipynb

```diff
@@ -519,26 +519,42 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Using Station with a YAML configuration file"
+    "### Using Station with YAML configuration files"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this section, we shall briefly describe the usage of `Station` with a YAML configuration file. For more details, we refer to the docstrings of the methods of the `Station` class.\n",
+    "In this section, we shall briefly describe the usage of `Station` with YAML configuration files. For more details, we refer to the docstrings of the methods of the `Station` class.\n",
     "\n",
-    "A `Station` with a YAML configuration file can be created by passing the file name (or file name with absolute path) to `Station`s constructor. File name and location resolution also takes into account related entries in the `'station'` section of the QCoDeS configuration. We refer to corresponding documentation for more information.\n",
+    "A `Station` can be created with or without one or more YAML configuration files. With YAML configuration files, `Station` can be created by passing the file names (or file names with absolute path) to `Station`s constructor. File names and location resolution also takes into account related entries in the `'station'` section of the QCoDeS configuration. We refer to corresponding documentation for more information.\n",
     "\n",
     "```python\n",
     "station = Station(config_file='qutech_station_25.yaml')\n",
     "```\n",
     "\n",
-    "Alternatively, `load_config_file` method can be called on an already instantiated station to load the configuration file.\n",
+    "or\n",
+    "\n",
+    "```python\n",
+    "station = Station(config_file=['example_1.yaml', 'example_2.yaml'])\n",
+    "```\n",
+    "\n",
+    "Alternatively, `load_config_file` or `load_config_files` methods can be called on an already instantiated station to load the configuration files.\n",
+    "\n",
+    "* `load_config_file` method can be used for loading 1 configuration file.\n",
+    "* `load_config_files` method can be used for loading 1 or more configuration files. If multiple configuration files are provided, this method first merges them into one and then loads.\n",
```

Comment:
this method first merges them into one and then loads.  --?--> this method first merges them into one temporary YAML file and then loads that into the station.